### ISSUE TITLE
Boot with Discrete Graphics Only Enabled?

### ISSUE THREAD

**Issue by cyberfunk** [codykrieger/gfxCardStatus#261]
The retina MacBook Pro Mid-2012 and El Capitan causes crashing and random restarts when Automatic Graphics Switching is enabled. The solution is to use Discrete graphics card only. Is it possible to set gfxCardStatus to Discrete only on boot?

**Comment by codykrieger** [comment]
You don't need gfxCardStatus for that. Just:

- Quit gfxCardStatus if it's running
- Move it to the Trash
- Go to System Preferences > Energy Saver
- Uncheck the “Automatic graphics switching” checkbox

Then your machine will only use the discrete GPU.

**Comment by cyberfunk** [comment]
Hi Cody,
I responded via email but was having issues with my outdated iPhone 4s. I was trying to say that on a MacBook Pro Retina 2012 under El Capitan Automatic Graphics Switching will re-enable itself. The only warning you get is after your computer crashes and you check system preferences to find that it's been re-enabled. I'm not sure how it's happening but the same issue was present on my old 2011 MacBook pro under Snow Leopard. When I used an old version of your app with memory settings it fixed my issues perfectly. Is it possible to re-introduce this feature with your current version 2.3+.
